site/static/docs/4.3/assets/img/favicons/safari-pinned-tab.svg

```diff
@@ -1,4 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="933.333" height="933.333" version="1" viewBox="0 0 700 700">
-  <path d="M104.5.7C89.2 2.6 77.4 6.2 63.8 13.2 41.7 24.5 24.5 41.7 13.2 63.8 7.5 75.1 4.8 82.3 2.3 94 .6 102.1.5 114.3.5 350s.1 247.9 1.8 256c2.5 11.7 5.2 18.9 10.9 30.2 11.3 22.1 28.5 39.3 50.6 50.6 11.3 5.7 18.5 8.4 30.2 10.9 8.1 1.7 20.3 1.8 256 1.8s247.9-.1 256-1.8c11.7-2.5 18.9-5.2 30.2-10.9 22.1-11.3 39.3-28.5 50.6-50.6 5.7-11.3 8.4-18.5 10.9-30.2 1.7-8.1 1.8-20.3 1.8-256s-.1-247.9-1.8-256c-2.5-11.7-5.2-18.9-10.9-30.2C670.9 32.6 642.9 11 607 2.4 599.9.7 587.2.6 353.5.4 218.2.3 106.2.5 104.5.7zm315.3 153.6c47.4 9 75.3 30.5 85.6 65.9 5.1 17.8 5.6 43.1 1.1 60.3-2 7.5-7.9 20.3-12.2 26.4-8 11.3-21.9 22.8-36 30-3.5 1.7-6.3 3.5-6.3 3.9 0 .5 2.3 1.4 5.1 2 2.8.7 8.5 2.6 12.7 4.3 37.2 14.8 58.1 50.4 58.2 99 0 28.6-9 53.9-25.7 71.8-18.8 20.3-45.2 32.9-83.3 39.8-8.5 1.5-20.9 1.7-119.2 2l-109.8.4V151.9l110.8.4c95.5.3 111.8.6 119 2z"/>
-  <path d="M262 266.5v56.6l65.3-.4c57.2-.3 65.9-.5 70.7-2 15.8-4.8 28.7-14.9 34.4-27.1 4.4-9.2 5.6-15.3 5.6-28.4-.1-25.1-7.5-39.8-24.3-47.7-14.1-6.7-14.2-6.7-86.4-7.2l-65.3-.5v56.7zm0 171.5v64.1l71.3-.3c69.8-.3 71.4-.4 79-2.6 11.4-3.2 19.2-7.7 27.2-15.7 12.1-12 16.8-24.9 16.8-46 0-20.8-5.1-34.3-17.3-45.9-7.9-7.4-15.8-11.6-28.1-14.7-8.1-2.1-10.5-2.2-78.6-2.6l-70.3-.5V438z"/>
+<svg xmlns="http://www.w3.org/2000/svg" width="192" height="192" fill="none" viewBox="0 0 192 192">
+  <path fill="#000" d="M0 0h192v192H0z"/>
+  <path fill="#fff" fill-rule="evenodd" d="M136 20H56c-19.882 0-36 16.118-36 36v80c0 19.882 16.118 36 36 36h80c19.882 0 36-16.118 36-36V56c0-19.882-16.118-36-36-36zm-80-4c-22.091 0-40 17.909-40 40v80c0 22.091 17.909 40 40 40h80c22.091 0 40-17.909 40-40V56c0-22.091-17.909-40-40-40H56z" clip-rule="evenodd"/>
+  <path fill="#fff" fill-rule="evenodd" d="M101.367 136H66.621V51.45h33.984c15.879 0 25.43 8.085 25.43 21.093 0 8.848-6.504 16.758-14.824 18.164v.996c11.425 1.172 19.043 9.317 19.043 20.332 0 14.883-10.84 23.965-28.887 23.965zM79.746 62.582V87.25h15.586c11.601 0 17.636-4.277 17.636-12.305 0-7.793-5.683-12.363-15.41-12.363H79.746zm18.984 62.285H79.746V97.621h18.222c12.364 0 18.868 4.629 18.868 13.477 0 8.964-6.27 13.769-18.106 13.769z" clip-rule="evenodd"/>
```

Comment:
Just tested running the .svg through https://jakearchibald.github.io/svgomg/ with `Precision: 2` we can save about 20% file size with no (visual) loss in detail

Reply:
Thanks, but I'm not interested in further SVG optimizations